validation-test/Evolution/Inputs/class_change_lazy_to_computed.swift

```diff
@@ -0,0 +1,33 @@
+
+#if BEFORE
```

Comment:
These are backwards, no? BEFORE should be the lazy one. (I mean, we're testing in both directions, but the names should match the code.)

Reply:
Done